src/core/models/outputs/output_alias.c

```diff
@@ -11,10 +11,13 @@
 
 // maximum number of feature blocks
 #define MAX_ALIAS_FEATURE_BLOCKS_COUNT 3
```

Comment:
Maximum number of feature blocks for an alias output is 2 (sender and metadata).